include/alpaka/meta/Filter.hpp

```diff
@@ -49,7 +49,7 @@ namespace alpaka
             {
                 using type =
                     typename std::conditional<
-                        TPred<T>::value,    // TODO: Remove '::value' when C++14 variable templates are supported.
+                        TPred<T>::value,
```

Comment:
Question: the `::value` is not removed as the deleted comment suggests, is it the intention?

Reply:
Yes, this was intended. The comment did not apply anymore.